riscv/mmu.cc

```diff
@@ -242,6 +242,11 @@ void mmu_t::load_slow_path(reg_t addr, reg_t len, uint8_t* bytes, xlate_flags_t
       load_slow_path_intrapage(len - len_page0, bytes + len_page0, access_info.split_misaligned_access(len_page0));
   }
 
+  while (len > 8) {
+    check_triggers(triggers::OPERATION_LOAD, addr, access_info.effective_virt, reg_from_bytes(8, bytes));
+    len -= 8;
+    bytes += 8;
```

Comment:
Rather than hardcoding `8` on each of these lines, please use `sizeof(reg_t)` to make the intent more obvious.